kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/SubjectAccessReviewDSLImpl.java

```diff
@@ -68,6 +68,11 @@ public SubjectAccessReview create(SubjectAccessReview... resources) {
     return new CreatableSubjectAccessReview().create(resources);
   }
 
+  @Override
+  public SubjectAccessReview create(SubjectAccessReview resource) {
+    return new CreatableSubjectAccessReview().create(resource);
+  }
+
```

Comment:
Although right now it doesn't really matter, reusing the method with the array signature will prevent bugs in the future in case one of both methods gets changed:
```suggestion
  @SuppressWarnings("squid:S3878")
  @Override
  public SubjectAccessReview create(SubjectAccessReview resource) {
    return create(new SubjectAccessReview[]{resource});
  }

```